openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -305,14 +306,14 @@ class DeactivateLogoutView(APIView):
     """
     POST /api/user/v1/accounts/deactivate_logout/
```

Comment:
I know that the view name and endpoint were already determined prior to this PR, but they both feel implementation-specific; what if we don't want to logout in the future (not saying we would, just trying to use a trivialized example)? 

Or if we also want to close the active window in the future - would / should we call this `DeactivateLogoutCloseActiveWindowView`?

It feels like the workflow this method is trying to represent is "simply" the act of deactivating / retiring an account, at least for the caller of this API. Maybe "CompleteDeactivation" or "CompleteRetirement" (though I'm not a fan of the "Complete" #namingishard) 

🤷‍♂️ 

Just a thought.

Reply:
Point taken, but I'm not going to relitigate the name of this endpoint when we are scheduled to get this into manual testing this week.

Reply:
I am going off of https://openedx.atlassian.net/wiki/spaces/PLAT/pages/618856813/REST+API+Endpoints+for+GDPR+User+Retirement and sticking to this plan is crucial given the cross-functional nature of this project.